### DISCUSSION TITLE
Split of forcing files in time

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/Wflow.jl#290] [Q&A] [answered]
If forcing files are split in time (Figure 1), then ending time (and days since from ...) of one file and starting time (and days from ...) of the next file should either be in increasing order as shown in Figure 2 (first file) and Figure 3 (second file) or each file should have its own start from zero?

Figure 1: 
<img width="676" alt="image" src="https://user-images.githubusercontent.com/130187246/232260205-2d4dd153-f5f2-413c-ac50-9e45159c9a70.png">

Figure 2:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/130187246/232260290-7c2544ee-eb80-409f-b8ac-cd024acd96cd.png">

Figure 3:
<img width="454" alt="image" src="https://user-images.githubusercontent.com/130187246/232260295-527304f2-d376-4bd4-89b4-d861d98762f4.png">

**Comment by JoostBuitink** [comment, marked as answer]
When using multiple forcing files, we expect the time dimension to be continuous over the separate files. For example, if you have a forcing timeseries from 1990 to 1991 split per year, you end up with two files. The first file has time values running from 1990-01-01 until 1990-12-31, and the second file starts from 1991-01-01 and continues until 1991-12-31. The 'days since' is used to interpret the dates from indices (`time=9` for example) into a datetime object.
